src/NzbDrone.Core/IndexerSearch/NzbSearchService.cs

```diff
@@ -257,6 +260,13 @@ private List<DownloadDecision> Dispatch(Func<IIndexer, IEnumerable<ReleaseInfo>>
 
             foreach (var indexer in indexers)
             {
+                var indexerStatus = _indexerStatusService.GetIndexerStatus(indexer.Definition.Id);
+                if (indexerStatus != null && indexerStatus.DisabledTill.HasValue && indexerStatus.DisabledTill.Value > DateTime.UtcNow)
```

Comment:
there is too much logic around whether or not an indexer is usable due to it's status, I think we should move this logic to `IndexerStatus` class as methods. `indexerStatus.IsDisabled()`